libs/ui/src/remove_card_page.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Main } from './main';
+import { Prose } from './prose';
+import { fontSizeTheme } from './themes';
+
+const RemoveCardImage = styled.img`
+  margin-right: auto;
+  margin-bottom: 5px;
+  margin-left: auto;
+  height: 20vw;
+`;
+
+export function RemoveCardPage(): JSX.Element {
+  return (
+    <Main padded centerChild>
+      <RemoveCardImage src="remove-card.svg" alt="Remove Card Icon" />
```

Comment:
I know we haven't been consistent about this in the frontends' `public/` directories, but I see we've begun trying to group public items. Let's accordingly move these images to [`public/assets/`](https://github.com/votingworks/vxsuite/tree/main/frontends/election-manager/public/assets)